Classes/Updates/v87/RowUpdater/ImageCropUpdater.php

```diff
@@ -27,7 +27,7 @@
  * using "l10n_mode" set to "mergeIfNotBlank".
  * @internal This class is only meant to be used within EXT:install and is not part of the TYPO3 Core API.
  */
-class ImageCropUpdater implements RowUpdaterInterface
+class ImageCropUpdater implements \TYPO3\CMS\Install\Updates\RowUpdater\RowUpdaterInterface
```

Comment:
This change would make the TYPO3\CMS\v87\Install\Updates\RowUpdater\RowUpdaterInterface obsolete and it should be removed.